File: buttfield-addision-manning/Photos/Photos/ViewController.m
//
//  ViewController.m
//  Photos
//
//  Created by Syd Polk on 7/11/13.
//  Copyright (c) 2013 Bone Jarring Games and Software. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)takePhoto:(id)sender {
    UIImagePickerController *picker = [[UIImagePickerController alloc] init];
    if ([UIImagePickerController isSourceTypeAvailable:UIImagePickerControllerSourceTypeCamera]) {
        picker.sourceType = UIImagePickerControllerSourceTypeCamera;
        if ([UIImagePickerController isCameraDeviceAvailable:UIImagePickerControllerCameraDeviceFront]) {
            picker.cameraDevice = UIImagePickerControllerCameraDeviceFront;
        } else {
            picker.cameraDevice = UIImagePickerControllerCameraDeviceRear;
        }
        picker.delegate = self;
        
        [self presentViewController:picker animated:YES completion:nil];
    }
}

- (IBAction)loadFromLibrary:(id)sender {
    UIImagePickerController *picker = [[UIImagePickerController alloc] init];
    picker.sourceType = UIImagePickerControllerSourceTypeSavedPhotosAlbum;
    
    picker.delegate = self;
    [self presentViewController:picker animated:YES completion:nil];
}

- (void)imagePickerController:(UIImagePickerController *)picker didFinishPickingMediaWithInfo:(NSDictionary *)info {
    UIImage *image = image;
    [picker dismissViewControllerAnimated:YES completion:nil];
}

- (void)imagePickerControllerDidCancel:(UIImagePickerController *)picker {
    [picker dismissViewControllerAnimated:YES completion:nil];
}

@end

File: gk-brown/HelloMarkupKit/HelloMarkupKit/ViewController.swift
//
//  ViewController.swift
//  HelloMarkupKit
//
//  Created by Syd Polk on 7/5/17.
//  Copyright © 2017 Bone Jarring Games and Software. All rights reserved.
//

import UIKit
import MarkupKit

class ViewController: UIViewController {
    
    @IBOutlet var greetingLabel: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        LMViewBuilder.view(withName: "ViewController", owner: self, root: view)
    }

    @IBAction func sayHello() {
        greetingLabel.text = "Hello, World!"
    }

}

